apps/meteor/app/api/server/v1/oauthapps.ts

```diff
@@ -37,7 +37,7 @@ API.v1.addRoute(
 			}
 
 			if ('appId' in this.queryParams) {
-				apiDeprecationLogger.parameter(this.request.route, 'appId', '7.0.0', this.response);
+				apiDeprecationLogger.parameter(this.route, 'appId', '7.0.0', this.response);
```

Comment:
Since you editing these, should we put 8.0 instead of 7?

Reply:
i think it is a job for the next task :p